openstack/kos-operator/templates/deployment.yaml

```diff
@@ -22,6 +22,8 @@ spec:
       name: kos-operator
   template:
     metadata:
+      annotations:
+      {{- include "utils.linkerd.pod_and_service_annotation" . | indent 8 }}
```

Comment:
I far, that needs to be an `nindent 8` so we get the newline back that the `{{-` eats

Reply:
I've added the commit. I hope you don't mind.

Reply:
(I was mistaken, it seems. The template itself adds a newline)